### PR TITLE
docs: add instructions to install the CLI through Krew

### PR DESCRIPTION
**Pull request by rbtr**
# Description

[Retina is available in Krew!](https://github.com/kubernetes-sigs/krew-index/pull/3722)
Updates the docs with installation instructions using Krew.
Also updates the Helm chart installation instructions to fetch the latest released version instead of hardcoding a version.

## Related Issue

related to #108 
provides a permanent fix for #163

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,10 +86,11 @@ For further CLI documentation, see [Capture with Retina CLI](https://retina.sh/d
...
 
 ```bash
+VERSION=$( curl -sL https://api.github.com/repos/microsoft/retina/releases/latest | jq -r .name)
 helm upgrade --install retina oci://ghcr.io/microsoft/retina/charts/retina \
-    --version v0.0.2 \
-    --set image.tag=v0.0.2 \
-    --set operator.tag=v0.0.2 \
+    --version $VERSION \
+    --set image.tag=$VERSION \
+    --set operator.tag=$VERSION \
```
Comment (timraymond) [inline README.md:93]: ❤️

**Review by timraymond — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -40,10 +40,12 @@ Retina has two major features:
 Retina can be installed using the Helm chart from GHCR:
 
 ```bash
+# Set the version to a specific version here or get latest version from GitHub API.
+VERSION=$( curl -sL https://api.github.com/repos/microsoft/retina/releases/latest | jq -r .name)
 helm upgrade --install retina oci://ghcr.io/microsoft/retina/charts/retina \
-    --version v0.0.2 \
-		--set image.tag=v0.0.2 \
-		--set operator.tag=v0.0.2 \
+    --version $VERSION \
```
Comment (nddq) [inline README.md:46]: is this possible in markdown? i didn't know that 🤔

Comment (rbtr) [reply README.md:46]: it's not dynamic, it's just for you to copy and paste

Comment (nddq) [reply README.md:46]: oops nvm lol

**Review by nddq — APPROVED** [review]

**Pull request merged**